Core/Goals/LootGoal.cs

```diff
@@ -227,6 +233,9 @@ private bool IsPlayerMoving(WowPoint lastPos)
 
         private async Task GoalExit()
         {
+            await input.TapLastTargetKey("Last target key");
+            await input.TapInteractKey("Interract key");
+            await Task.Delay(5000);
```

Comment:
The loot should not happen at this time frame. 

This function servers as the LootGoal as been finished either successfully or not and notify the other Goals the `GoapKey.shouldloot` has been changed to `false`

Reply:
it's a crutch